routes/contactUs/helpers.js

```diff
@@ -0,0 +1,26 @@
+import emailJs from "emailjs";
+
+export const contactUsmail = async (req, res) => {
+    try {
+        const server = await emailJs.server.connect({
+            host:    "smtp.gmail.com",
+            password:"agroeasy1$##",
+            ssl:     true,
+            user:    "agroeasy2018@gmail.com",
```

Comment:
You probably don't want to save the password to your business email account in a public repository. Not probably, definitely

Reply:
`host` and `user` strings should be saved in constants file

Reply:
i think dot env will be cool here

Reply:
yeah that works just as long as you dont commit it to the repo. You can do what `tribl` is doing by having an example `.env` file